lib/presentation/widget/myspace/my_space_widget.dart

```diff
@@ -99,6 +121,78 @@ class _MySpaceWidgetState extends State<MySpaceWidget> {
         converter: (Store<AppState> store) => mySpaceViewModel);
   }
 
+  Widget _buildMySpaceList(BuildContext context) {
+    return StoreConnector<AppState, List<Document>>(
+        converter: (store) => store.state.mySpaceState.documentList,
+        builder: (context, documentList) {
+          if (documentList.isEmpty) {
+            return _buildUploadFileHere(context);
+          } else {
+            return ListView.builder(
+                itemCount: documentList.length,
+                itemBuilder: (context, index) {
+                  return _buildMySpaceListItem(context, documentList[index]);
+                });
+          }
+        });
+  }
+
+  Widget _buildMySpaceListItem(BuildContext context, Document document) {
+    return ListTile(
+      leading: Image.asset(
+        imagePath.icFileTypeImage,
+        width: 16,
+        height: 20,
+        fit: BoxFit.fill,
+      ),
+      title: Transform(
+        transform: Matrix4.translationValues(-16, -2, 0.0),
+        child: Text(
+          document.name,
+          maxLines: 1,
+          style:
+              TextStyle(fontSize: 14, color: AppColor.documentNameItemTextColor),
+        ),
+      ),
+      subtitle: Transform(
+        transform: Matrix4.translationValues(-16, 0.0, 0.0),
+        child: Row(
+          children: [
+            Text(
+              'Modified ${document.modificationDate.getMMMddyyyyFormatString()}',
```

Comment:
I will change `Modified`text later after @hoangdat PR is merged.

Reply:
you will use `Intl`?

Reply:
yes I will